oidc-consumer/src/interfaces/index.ts

```diff
@@ -2,6 +2,7 @@ import { Session, SessionData, SessionOptions } from "express-session";
 import { ModuleOptions } from "simple-oauth2";
 
 export interface IConsumerOptions {
+  timeout: number;
```

Comment:
better to accept this within session-options as it only effects that
could be interpreted otherwise if it's a top-level prop

Reply:
But we use express-session's session-options. Can we pass custom options there?